pyvista/plotting/charts.py

```diff
@@ -1374,40 +1371,33 @@ def show(
 
         Parameters
         ----------
-        off_screen : bool, optional
+        off_screen : bool, default: pyvista.global_theme.off_screen
             Plots off screen when ``True``.  Helpful for saving screenshots
-            without a window popping up.  Defaults to active theme setting in
-            :attr:`pyvista.global_theme.full_screen
-            <pyvista.themes.DefaultTheme.full_screen>`.
+            without a window popping up. Defaults to active theme setting.
 
-        full_screen : bool, optional
+        full_screen : bool, default: pyvista.global_theme.full_screen
             Opens window in full screen.  When enabled, ignores
```

Comment:
This example is one where I think it makes sense to not use `optional`, but it makes the documentation inconsistent.  I'm +0.5 for making this a documentation practice when `None` is immediately replaced by the global theme.

Reply:
Good point. Let's keep it as is when the additional changes it not needed.